src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs

```diff
@@ -247,6 +243,245 @@ IEnumerable<int> Goo()
 }");
         }
 
+        [Fact]
+        public void GetBreakpointSequence7()
+        {
+            TestAll(@"
+class C
+{
+    IEnumerable<int> Goo()
+    $$[|{|]
+        [|_ = M2(
+            c
+            switch
+            {
+                (1) _ [|when f|] => [|M(0)|],
+                (3) _ [|when f|] => [|M(1)|],
+                (1, 2) _ [|when f|] => [|M(0)|],
+                (3, 4) _ [|when f|] => [|M(2)|],
+                _ => [|M(4)|],
+            },
+            M(5));|]
+    [|}|]
+}");
+        }
+
+        [Fact]
+        public void SwitchExpression01()
+        {
+            TestSpan(
+@"class C
+{
+    void Goo()
+    {
+$$        [|_ = e switch
+        {
+            1 when f => 2,
+            3 when g => 4,
+            _ => 5,
+        };|]
+    }
```

Comment:
Shouldn't there be a sequence point on the closing brace here? There's one in the method above

Reply:
Wasn't planning to have one.  This closing brace is part of the syntax of the switch expression, not the end of a block.

---
In reply to: [401232766](https://github.com/dotnet/roslyn/pull/42711#discussion_r401232766) [](ancestors = 401232766)